src/hdmf_zarr/backend.py

```diff
@@ -336,15 +463,17 @@ def write_group(self, **kwargs):
                 self.write_group(parent=group,
                                  builder=sub_builder,
                                  link_data=link_data,
-                                 exhaust_dci=exhaust_dci)
+                                 exhaust_dci=exhaust_dci,
+                                 number_of_jobs=number_of_jobs)
 
         datasets = builder.datasets
         if datasets:
             for dset_name, sub_builder in datasets.items():
                 self.write_dataset(parent=group,
                                    builder=sub_builder,
                                    link_data=link_data,
-                                   exhaust_dci=exhaust_dci)
+                                   exhaust_dci=exhaust_dci,
+                                   number_of_jobs=number_of_jobs)
```

Comment:
Should these pass the 'max_threads_per_process' and 'multiprocessing_context' kwargs?

Reply:
See above